### PR TITLE
User is able to see their score after the game is completed.

### PR DESCRIPTION
**Pull request by JohnRatcliffe209**

### CODE REVIEW

File: Public/Script/modeoptions.js [hunk 1]
```diff
@@ -1,6 +1,11 @@
 'use strict'
 
-const button = document.getElementById('singlePlayerMode')
-button.addEventListener('click', function () {
-  location = String(location).replace("modeoption", "hostlobby")
+const buttonSingle = document.getElementById('singlePlayerMode')
+buttonSingle.addEventListener('click', function () {
+  location = String(location).replace('modeoption', 'hostlobby')
```
Comment (lethiwe6) [inline Public/Script/modeoptions.js:5]: I think for single player it should be replace('modeoption','Singleplayer') , single player doesnot require the lobby

**Review by lethiwe6 — COMMENTED** [review]
Everything looks good

**Comment by Thendo2000** [comment]
Looks good,I like the added comments to better explain how the scoring works and how many points are assigned to each colour.

**Review by tshililopercy — COMMENTED** [review]
The Scoring system works for wins anywhere between the board by doesn't processed evaluation for the last row... 

When the player wins, it shows the message before coloring the Words letters, please check these things out.

**Review by tshililopercy — APPROVED** [review]

**Review by Thendo2000 — APPROVED** [review]
Aside from one issue ,everything is good

**Pull request merged**